content/sensu-go/6.9/operations/manage-secrets/secrets-management.md

```diff
@@ -125,7 +131,196 @@ EOF
+
+In the Conjur Docker container, run the following command to create a sensu.io account and initialize the admin user:
+
+{{< code shell >}}
+docker-compose exec conjur conjurctl account create sensu.io
+{{< /code >}}
+
+The response should confirm that Conjur created the sensu.io account and include an admin API key.
+For example:
+
+{{< code text >}}
+Created new account 'sensu.io'
+Token-Signing Public Key: -----BEGIN PUBLIC KEY-----
+MIIBIjANB...
+-----END PUBLIC KEY-----
+API key for admin: 5pj0kdx56npm8gksd9gre69sb21vh7zws2nk8jy73ekcyjk8e0xdj3
+{{< /code >}}
+
+### Write and load a Conjur policy with the secrets variable and host
+
+Use the [Conjur CLI][39] to load a policy and secrets for Sensu to use.
+
+1. Initialize the Conjur CLI client to use the sensu.io account and point it at the correct URL:
+
+   {{< code shell >}}
+conjur init -u conjur -a sensu.io
+{{< /code >}}
+
+2. Write a Conjur policy that defines a human user, a non-human identity that represents a Sensu backend, at least one secret variable, permissions for the human user, and a host for the Sensu backend to use:
+
+   {{< code shell >}}
+cat << EOF > /tmp/policy.yaml
+- !policy
+  id: Sensu
+  body:
+  - !user SensuAdmin
+  - !host sensuBackend
+  - !variable
+    id: pagerDutyAPIKey
+  - !permit
+    role: !user SensuAdmin
+    privileges: [read, update, execute]
+    resource: !variable pagerDutyAPIKey
+  - !permit
+    role: !host sensuBackend
+    privileges: [read, execute]
+    resource: !variable pagerDutyAPIKey
+EOF
+{{< /code >}}
+
+3. Load the policy:
+
+   {{< code shell >}}
+conjur policy load root /tmp/policy.yaml
+{{< /code >}}
+
+   {{% notice note %}}
+**NOTE**: Conjur will prompt for your username and password.
+Enter the username (`admin`) and password (the API key for admin) for the sensu.io account.
+This information is included in the response to the command to [create a Conjur account](#create-a-conjur-account).
+{{% /notice %}}
+
+   The response should include a list of created roles and credentials, similar to this example:
+
+   {{< code text >}}
+Loaded policy 'root'
+{
+  "created_roles": {
+    "sensu.io:user:SensuAdmin@Sensu": {
+      "id": "sensu.io:user:SensuAdmin@Sensu",
+      "api_key": "25xjdpm21p3obl2yw7w4d1xmvzj32psjbps3ftaj1e32rb94y3q3xge1"
+    },
+    "sensu.io:host:Sensu/sensuBackend": {
+      "id": "sensu.io:host:Sensu/sensuBackend",
+      "api_key": "2j78g5k0fb56fafsdoslehsaf246f092wmwkvh1qrgchk1tt8x8c"
+    }
+  },
+  "version": 1
+}
+{{< /code >}}
+
+   Copy and save the API key value for the sensu.io:host:Sensu/sensuBackend role &mdash; you will need it when you create your CyberArkProvider secrets provider in Sensu.
+
+4. Use the Conjur CLI to set the pagerDutyAPIKey secret, replacing `<INTEGRATION_KEY>` with your PagerDuty Integration Key:
+
+   {{< code shell >}}
+conjur variable values add Sensu/pagerDutyAPIKey <INTEGRATION_KEY>
+{{< /code >}}
+
+### Create your Conjur secrets provider
+
```

Comment:
I could not test the rest of the doc from this point on. The backend won't stay up in the skunkworks scenario that builds from develop_6.

Reply:
Thanks! I'm not sure if it's b/c I'm running Conjur at the same time or if it's an issue in develop_6.

Reply:
Thanks @c-kruse -- I've got it working now!